scripts/conda.sh

```diff
@@ -0,0 +1,9 @@
+echo "Creating conda env..."
+conda create -n epispot python=3.9
+echo "Activating env..."
+conda activate epispot
+echo "Completed!"
+echo "Installing requirements..."
```

Comment:
```suggestion
echo "Installing requirements with `pip`..."
```
Tell the user that the requirements are being installed via `pip` instead of `conda` to facilitate dependency management.